static/src/javascripts/lib/$.js

```diff
@@ -9,7 +9,7 @@ bonzo.aug({
     },
 });
 
-const $ = (selector: ?string | ?Node, context: ?Element | ?string): bonzo =>
+const $ = (selector: ?string | ?Node, context: ?Node | ?string): bonzo =>
```

Comment:
These are perfectly valid qwery calls, sadly:
```
qwery(window)
qwery(document)
qwery(document.createElement('x'))
qwery([1, 2, 3, [4, 5]])
```
... but we don't want any of that. IMHO the signature should be `(string, ?Element | ?Document): Element[]`.